[F10-4] MeNUS

azhikai opened the pull request:
@rebstan97 @m4dkip @yican95 @HyperionNKJ 
Closed #3 due to invalidity.

dalessr reviewed (commented):
Good job 👍 For now all the changes are fine. Can start to update user guide (i.e. features for each version) and developer guide (i.e. user story, use cases, NFR...) accordingly.

azhikai commented:
@dalessr Thank you. Yes, we have started doing that but not quite complete yet. 👍

damithc commented:
Good job on following the guidelines on team member photos closely. Your team is one of the few to have done it so far. 👍

dalessr reviewed (commented):
Good job 👍 @AZhiKai @rebstan97 @m4dkip @yican95 @HyperionNKJ

Some comments:
- User Guide: add description and examples for all the features due to consistency (i.e. help). Can omit examples of trivial features (i.e. list). Make the format of the tips consistent (can follow the format in 5.2). Some command words are quite long (i.e. list-ingredients-low), maybe can think of some shorter ones easier for users to remember? Also, remember to add "Coming in v2.0" for features not yet implemented.
- Developer Guide: can reorder user stories so that all high level features are displayed on top, followed by medium level features and low level features. Some use cases are incorrect. Remember that the steps in MSS reflect the interaction between user and system, which indicates that 2 system actions cannot appear consecutively.
- Also, remember to resolve conflicts in developer guide.

azhikai commented:
@dalessr Thank you on behalf of my team for the feedback. We will work on them for the upcoming milestone. 👍 
For the conflict, I think it's a conflict between our team repo's version and the AB4's version though, because on our end there's no conflict at all.

dalessr commented:
> For the conflict, I think it's a conflict between our team repo's version and the AB4's version though, because on our end there's no conflict at all.

Yup. Ok the conflict here is actually quite trivial, so can just resolve it because I think probably that is the reason why Travis CI test is not triggered in this PR.

azhikai commented:
Hi @dalessr , could you advice us on how to go about resolving the conflicts? We don't have write access to this repo and can't figure a way out.

dalessr commented:
Hmm can you see the [web editor](https://github.com/nus-cs2103-AY1819S1/addressbook-level4/pull/73/conflicts) link above `Conflicting files`? Can figure out the conflict first and do a commit on your org's `master` branch.

azhikai commented:
![image](https://user-images.githubusercontent.com/7473075/46612287-ab7a5000-cb42-11e8-9a49-01647f19a1ee.png)
This is what I'm seeing.

dalessr commented:
Uhh ok I see what happens. Never mind can just leave it. I will go directly to your team's repo to check Travis Build.

azhikai commented:
> Uhh ok I see what happens. Never mind can just leave it. I will go directly to your team's repo to check Travis Build.

Thanks. Please let us know if there's anything we need to do to resolve this conflict. 👍

azhikai commented:
Hi @dalessr , not sure why but Netlify failed to deploy here. Sometimes it happened in our team org repo, but redeploying it fixes the issue. Also, still not sure why Travis isn't triggered.

Do you think it's better if we open a new PR or something? Kindly advice, thanks! 😄

dalessr commented:
@AZhiKai Sorry for the late reply. Netlify sometimes fails in other PRs also, seems that it is not stable.

Travis cannot be triggered until the conflict is resolved. Since you have write access to your team's repo, can you try to merge the module's repo `master` branch into your team's repo `master` branch to resolved it?

azhikai commented:
> @AZhiKai Sorry for the late reply. Netlify sometimes fails in other PRs also, seems that it is not stable.
> 
> Travis cannot be triggered until the conflict is resolved. Since you have write access to your team's repo, can you try to merge the module's repo `master` branch into your team's repo `master` branch to resolved it?

Will do that after submitting v1.2. 👍

azhikai commented:
@dalessr Fixed the conflict.

dalessr reviewed (commented):
@AZhiKai @rebstan97 @m4dkip @yican95 @HyperionNKJ Good job for v1.2 👍 Some comments:

README:
- Remove the second point as this is an actual product designed for restaurant owners instead of se learning purpose.

User Guide:
- Can put `Coming in v2.0` features at the bottom of the command list
- Should add command format and example for 5.43 also, even though you will not be implementing it.

Developer Guide:
- Update/add diagrams where applicable. i.e.:
![image2](https://user-images.githubusercontent.com/22449685/47214426-7e466100-d3d0-11e8-88fa-f3f5d541be98.png)
(In your project, `Person` should become food item right?)
- Add labels for all the newly added diagrams (i.e. the sequence diagrams in implementation section)

azhikai commented:
@dalessr Thank you for the review on behalf of the team.

We have noted your feedback and will make the necessary changes accordingly. 

For `Person`, we're intending to remove it in v1.3 as it's not used at all.

azhikai commented:
@dalessr  We have began refactoring, may I check with you if it's okay if we don't refactor the package name? i.e. let it remain as `seedu.address`. But everything else has been refactored accordingly (no more mentions of address book, etc).

dalessr commented:
Sure. It should be fine as long as `address book` is not showing in the documentations (user guide, developer guide, readme, about us...) as well as the product.

azhikai commented:
> Sure. It should be fine as long as `address book` is not showing in the documentations (user guide, developer guide, readme, about us...) as well as the product.

Yup, everything related to "address book" has been removed from the code, UGDG, README, basically everywhere we can think of. 

Thanks for the clarification! 👍

azhikai commented:
@dalessr Hi Shengran, for the releases, is it okay if we leave the "description" as blank? As seen https://github.com/CS2103-AY1819S1-F10-4/main/releases

Ideally in real scenario we don't, but was wondering if the description in releases play a part when comes to grading.

dalessr commented:
> Ideally in real scenario we don't, but was wondering if the description in releases play a part when comes to grading.

It is fine to leave it blank, as only the jar file is required. We won't assess the descriptions in the release.

azhikai commented:
@dalessr  Kindly review our diagrams, should have already updated majority of it for v1.3.
Please ignore the figure numbers first, we forgot to change it to manual indication.

dalessr commented:
@AZhiKai Hmm.. not sure why netlify is not deployed. Can try to investigate on that or ask on slack?

azhikai commented:
> @AZhiKai Hmm.. not sure why netlify is not deployed. Can try to investigate on that or ask on slack?

@dalessr Hmm didn't realise that. I checked some other PRs and realise Netlify isn't deploy too. Will ask in Slack.

azhikai commented:
@damithc  Hi Prof, as mentioned on Slack, Netlify isn't triggered here.

damithc commented:
Seems to be working now?
![image](https://user-images.githubusercontent.com/1673303/47888071-69aca300-de7d-11e8-8935-5e1a297fff86.png)

azhikai commented:
> Seems to be working now?
> ![image](https://user-images.githubusercontent.com/1673303/47888071-69aca300-de7d-11e8-8935-5e1a297fff86.png)

@damithc  Seems like it took a new commit to trigger it. Apologies for the false alarm.

dalessr reviewed (commented):
The diagrams in dg look good to me in general 👍 Good job!

Just one question:
![screenshot20181102](https://user-images.githubusercontent.com/22449685/47948689-8bd31d80-df70-11e8-9107-705efdf208eb.png)
Here what does the action do? Is it calling method inside the class? If true, should it be drawn like that?

rebstan97 commented:
@dalessr Thank you for your feedback. Yes that is a mistake, it should return first before calling the method `parse()` and is not calling a method inside its own class. Will make the correction 👍

azhikai commented:
@dalessr  Can I confirm with you that our v1.4 submission is on 12/11/18 2359pm (week 13, monday)?

And also, what is the consequence of not removing `Person` stuff from the code base? Assuming we removed all the Person related commands from UG/DG, but you can still execute it. Reason is I'm not sure if we have enough time to fully remove everything, and still fix the test cases by the time of submission.

dalessr commented:
@AZhiKai Sorry for the late reply!

> Can I confirm with you that our v1.4 submission is on 12/11/18 2359pm (week 13, monday)?

Yes. This is what I found on website:
- **Deadline** for all v1.4 submissions is Week 13 Monday 23.59 unless stated otherwise.

> And also, what is the consequence of not removing Person stuff from the code base? Assuming we removed all the Person related commands from UG/DG, but you can still execute it. Reason is I'm not sure if we have enough time to fully remove everything, and still fix the test cases by the time of submission.

It it ok to leave the original code, though in real software development this should be avoided. As long as you can implement your own features correctly with reasonable amount of documentation and tests, it is good enough :-p

azhikai commented:
> @AZhiKai Sorry for the late reply!
> 
> > Can I confirm with you that our v1.4 submission is on 12/11/18 2359pm (week 13, monday)?
> 
> Yes. This is what I found on website:
> 
> * **Deadline** for all v1.4 submissions is Week 13 Monday 23.59 unless stated otherwise.
> 
> > And also, what is the consequence of not removing Person stuff from the code base? Assuming we removed all the Person related commands from UG/DG, but you can still execute it. Reason is I'm not sure if we have enough time to fully remove everything, and still fix the test cases by the time of submission.
> 
> It it ok to leave the original code, though in real software development this should be avoided. As long as you can implement your own features correctly with reasonable amount of documentation and tests, it is good enough :-p

Okay, thanks for the clarification.

For the latter part, I have begun removing the code, probably 80% there already. Really quite tedious imo...no wonder morphing is a "high risk high reward" option. 😓

azhikai commented:
Hi @dalessr 

For Javadocs, do we need to include the `@params`, `@throws`, etc? I realise that the existing AB4 code base ignores it, and basically just gives a description.

dalessr reviewed (commented):
The documentations LGTM 👍 Good job everyone @AZhiKai @rebstan97 @m4dkip @yican95 @HyperionNKJ

> For Javadocs, do we need to include the @params, @throws, etc? I realise that the existing AB4 code base ignores it, and basically just gives a description.

It's good to have if you feel that it will help other developers to understand the implementation. If the method is trivial (and only used once internally), then it's ok to ignore or just provide a brief description.

yican95 committed:
Merge branch 'master' of https://github.com/yican95/main into docs/PPP

azhikai committed:
docs: Update UI component pptx and png

Currently it still display Person-related UI.

This might confuse future developers as it is the only remaining portion of the developer guide that still mentions "Person".

Let's update it to fit the existing state of the application.

azhikai committed:
Merge branch 'master' into hotfix/readme-dg

# Conflicts:
#	docs/DeveloperGuide.adoc

yican95 committed:
UserGuide.adoc: Update menu commands

yican95 committed:
DeveloperGuide.adoc: Update menu implementation

m4dkip committed:
Update PPP

m4dkip committed:
Update User Guide

yican95 committed:
Update PPP

HyperionNKJ committed:
Improve example for record-sales

yican95 committed:
UserGuide.adoc: Update macros

azhikai committed:
DeveloperGuide: Update text

rebstan97 committed:
Update edit-ing to include field on/ if edit by name

HyperionNKJ committed:
Issue #286 bugfix

rebstan97 committed:
Update tests

rebstan97 committed:
Remove unused imports

rebstan97 committed:
Fix checkstyle errors

rebstan97 committed:
Fix checkstyle error

rebstan97 committed:
Remove MenuCommandTestUtil

rebstan97 committed:
Add stockUp test in RestaurantBookParser

rebstan97 committed:
Fix wrong import order

rebstan97 committed:
Rename variables with NUMUNITS to NUM_UNIT

rebstan97 committed:
Add code annotation

rebstan97 committed:
Fix errors

rebstan97 committed:
Fix errors

HyperionNKJ committed:
Fix checkstyle error

HyperionNKJ committed:
Change internal representation of records in SalesReport

azhikai committed:
Merge pull request #284 from AZhiKai/hotfix/readme-dg

Update README and DG

yican95 committed:
Update PPP

rebstan97 committed:
Add annotations for test code

yican95 committed:
Update PPP

rebstan97 committed:
Add tests to improve code coverage

rebstan97 committed:
Update userguide

rebstan97 committed:
Merge branch 'master' into Bug-fixes

HyperionNKJ committed:
Add model null checks, Update javadocs in Model

azhikai committed:
team/azhikai: Update PPP

yican95 committed:
Update PPP

azhikai committed:
Rename class name

HyperionNKJ committed:
Add test for RecordListPanel

HyperionNKJ committed:
Merge branch 'master' into bugfix/issue286

azhikai committed:
FindAccountCommand: Add command class

azhikai committed:
HelpCommand: Remove full stop

Since all the other commands' success message does not end with any punctuation, let's standardize it by removing the full stop here.

HyperionNKJ committed:
Merge pull request #287 from HyperionNKJ/bugfix/issue286

Fix detail panel not showing when previously selected record/ingredient is selected again

rebstan97 committed:
Merge branch 'master' into docs/PPP

# Conflicts:
#	docs/UserGuide.adoc
#	docs/team/rebstan97.adoc

HyperionNKJ committed:
Merge branch 'master' into code/annotate

rebstan97 committed:
Update PPP

azhikai committed:
FindAccountCommandParser: Accept command

m4dkip committed:
Fix typo in PPP

m4dkip committed:
Merge branch 'master' of https://github.com/m4dkip/main into updatePPP

yican95 committed:
Update PPP

m4dkip committed:
Update UG

yican95 committed:
DeveloperGuide.adoc: Add figure reference

m4dkip committed:
Update DG with figure numbers

rebstan97 committed:
Update user guide

rebstan97 committed:
Merge branch 'master' into Bug-fixes

azhikai committed:
FindAccountCommand: Update predicate to check for substring

Currently, the predicate that searches if a username contains a keyword only passes if there's a full word match. For example, testing if "azhikai" contains "kai" will return false.

This is not the intended behavior since username cannot contain any spaces.

Let's fix this behavior by checking if it contains a substring, rather than the whole word. That means testing if "azhikai" contains "kai" will now return true.

azhikai committed:
UsernameContainsKeywordPredicate: Update javadoc

yican95 committed:
Update menu diagrams

azhikai committed:
UsernameContainsKeywordPredicateTest: Add more tests

yican95 committed:
DeveloperGuide.adoc: Remove export menu usecase

rebstan97 committed:
Update version 2.0 features and command history

rebstan97 committed:
Merge pull request #288 from rebstan97/Bug-fixes

Bug fixes and enhancements

azhikai committed:
FindAccountCommandTest: Add tests

rebstan97 committed:
Merge branch 'master' into docs/PPP

# Conflicts:
#	docs/UserGuide.adoc

azhikai committed:
RestaurantBookParserTest: Add test for find-account command

rebstan97 committed:
Fix error in formatting

HyperionNKJ committed:
Change representation of Price from float to double

rebstan97 committed:
Update command summary

m4dkip committed:
Update Figure numbers

HyperionNKJ committed:
Change internal representation of ReportBuilder

HyperionNKJ committed:
Update commands

azhikai committed:
UserGuide: Update find-account section

yican95 committed:
UserGuide.adoc: Update command summary

azhikai committed:
FindAccountCommand: Catches exception if multiple keywords provided

Currently, if user provides multiple keywords, an exception will be thrown and the application will crash.

This behavior should not be intended as the command only accepts one keyword.

Let's fix this bug by catching the exception, and showing an appropriate message to the user.

yican95 committed:
DeveloperGuide.adoc: Add figure number

azhikai committed:
FindAccountCommandTest: Improve test

azhikai committed:
FindAccountCommandTest: Update comment

azhikai committed:
FindAccountCommandParserTest: Add tests for parser

yican95 committed:
UserGuide.adoc: Fix typo

rebstan97 committed:
Cropped sequence diagrams for delete ingredient feature

yican95 committed:
DeveloperGuide.adoc: Update menu implementation

yican95 committed:
Update menu implementation

yican95 committed:
Update PPP

m4dkip committed:
Update UG (Capitalization)

yican95 committed:
Update PPP

azhikai committed:
docs/diagram: Update sequence diagram

Updated the diagram due to improvement to the command.

azhikai committed:
docs/images: Update user session activity diagram

Updated the diagram due to change in how the logic is handled.

yican95 committed:
Fix typo

yican95 committed:
DeveloperGuide.adoc: Fix typo

m4dkip committed:
Update UG (Remove fullstops)

rebstan97 committed:
Remove full stops for notes and warnings

azhikai committed:
docs/diagrams: Fix storage component

Added missing navigability.

HyperionNKJ committed:
Add assert messages

HyperionNKJ committed:
Merge branch 'master' into code/annotate

azhikai committed:
team/azhikai: Update PPP

azhikai committed:
UserGuide: Update command format note

azhikai committed:
DeveloperGuide: Improve account section

azhikai committed:
Merge branch 'master' into feature/find-account

azhikai committed:
UserGuide: Increase UI highlight image size

m4dkip committed:
Update PPP (New PR links)

azhikai committed:
team/azhikai: Shift minor enhancement section

azhikai committed:
Merge pull request #292 from AZhiKai/feature/find-account

v1.4: Find account

HyperionNKJ committed:
Update comments

HyperionNKJ committed:
Merge branch 'master' into code/annotate

azhikai committed:
UserGuide: Update key feature section

azhikai committed:
team/azhikai: Improve language

azhikai committed:
Merge branch 'master' into feature/ug-dg-ppp

# Conflicts:
#	docs/UserGuide.adoc

yican95 committed:
Logic: Fix typo

yican95 committed:
UserGuide.adoc: Fix typo

azhikai committed:
UserGuide: Remove punctuation marks

m4dkip committed:
Update PPP (Update links)

HyperionNKJ committed:
Update UG

m4dkip committed:
Update PPP (Add credits to Natty)

yican95 committed:
Update PPP

HyperionNKJ committed:
RecordSalesCommand: Rephrase messages

HyperionNKJ committed:
Fix checkstyle error

yican95 committed:
Logic: Update invalid command message

azhikai committed:
DeveloperGuide: Add activity diagram for password proceessing

HyperionNKJ committed:
Merge pull request #293 from HyperionNKJ/code/annotate

v1.4: Annotate code and minor changes

m4dkip committed:
Update DG (Diagrams)

azhikai committed:
docs/diagrams: Update user session activity diagram

azhikai committed:
DeveloperGuide: Improve sentence phrasing

yican95 committed:
Update PPP

azhikai committed:
UserGuide: Rephrase introduction

yican95 committed:
Merge branch 'master' of https://github.com/yican95/main into docs/PPP

azhikai committed:
team/azhikai: Update PPP

azhikai committed:
UserGuide: Remove alias from command summary

To standardize with the rest, also to prevent confusion as the way I placed it made it looks like it might be a prefix.

rebstan97 committed:
Update UG

HyperionNKJ committed:
Update UG

HyperionNKJ committed:
Merge branch 'master' into docs/UGDGPPP

HyperionNKJ committed:
Update UG

HyperionNKJ committed:
Update UG

HyperionNKJ committed:
Update UG

HyperionNKJ committed:
Update UG

HyperionNKJ committed:
Update UG

HyperionNKJ committed:
Update DG

HyperionNKJ committed:
Update DG

HyperionNKJ committed:
Add figure number

HyperionNKJ committed:
Remove full stop

yican95 committed:
Model: Update message constraints

yican95 committed:
Model:Update message constraints

yican95 committed:
Logic: Update message usage

yican95 committed:
Logic: Update message usage

yican95 committed:
Logic: Update message usage

yican95 committed:
UserGuide.adoc: Update menu commands

rebstan97 committed:
Update DG

rebstan97 committed:
Merge branch 'master' into docs/PPP

HyperionNKJ committed:
Update PPP

rebstan97 committed:
Update DG

rebstan97 committed:
Update DG

yican95 committed:
Logic: Update DeleteItemByName to show all items after deleting item

m4dkip committed:
Update DG (Add transparency to images)

m4dkip committed:
Update DG (More information on Natty parsing date)

m4dkip committed:
Update DG (New parameters)

rebstan97 committed:
Update DG

rebstan97 committed:
Update PPP

yican95 committed:
Update UG & DG

azhikai committed:
Change "Inventory" to "Ingredients"

azhikai committed:
Merge branch 'master' into feature/ug-dg-ppp

yican95 committed:
Test: Add JUnit test

azhikai committed:
docs/diagrams: Fix sequence diagram error

yican95 committed:
Update message

azhikai committed:
DeveloperGuide: Add login for instructions for manual test

azhikai committed:
Remark: Privatize data and utilize toString()

yican95 committed:
Update message

yican95 committed:
Update message

yican95 committed:
UserGuide.adoc: Update menu commands

m4dkip committed:
Model: Update comment

m4dkip committed:
Update DG (Model diagram)

azhikai committed:
SampleDataUtil: Add method to get sample data

Currently, the sample data only contains the root account and nothing else.

This might be an issue when it comes to PE or product demonstration.

Let's update the file by including a method that populates the storage with decent amount of data. In addition, update related test cases (mainly the Clear-related cases) to use the method that generates only the root account and not the sample data.

yican95 committed:
Logic: Add message for DiscountItemCommand

azhikai committed:
DeveloperGuide: Add saving data, log out and change password to MT

yican95 committed:
Update UG & DG

yican95 committed:
Update UG & DG

yican95 committed:
Update UG & DG

azhikai committed:
DeveloperGuide: Include all commands in MT

yican95 committed:
DeveloperGuide.adoc: Update menu implementation

m4dkip committed:
Update DG (Instructions for Manual Testing)

azhikai committed:
SampleDataUtil: Add more data

m4dkip committed:
Update PPP (New contribution)

yican95 committed:
Update PPP, UG & DG

m4dkip committed:
Merge branch 'master' of https://github.com/m4dkip/main into updatePPP

azhikai committed:
DeveloperGuide: Update use case

m4dkip committed:
Update DG and PPP (Language and tagging)

azhikai committed:
FindAccountCommand: Update message usage

azhikai committed:
team/zhikai: Update PPP

azhikai committed:
DeveloperGuide: Update user session note

m4dkip committed:
Merge pull request #285 from m4dkip/updatePPP

v1.4 Update UG, DG, PPP

azhikai committed:
DeveloperGuide: Fix line break

azhikai committed:
Merge branch 'master' into hotfix/issue-295

yican95 committed:
Merge branch 'master' of https://github.com/yican95/main into docs/PPP

# Conflicts:
#	docs/DeveloperGuide.adoc

azhikai committed:
Merge branch 'master' into feature/ug-dg-ppp

# Conflicts:
#	docs/DeveloperGuide.adoc

azhikai committed:
Merge pull request #296 from AZhiKai/hotfix/issue-295

Hotfix: Include sample data for PE and PD

azhikai committed:
docs/diagram: Update model to include Name for account

azhikai committed:
team/zhikai: Update title

yican95 committed:
Update DG & PPP

azhikai committed:
team/zhikai: Rephrased sentence and bold sub titles

azhikai committed:
UserGuide: Change TIP to NOTE

yican95 committed:
Merge branch 'master' of https://github.com/yican95/main into docs/PPP

rebstan97 committed:
Add instructions for manual testing

azhikai committed:
team/azhikai: Add point to documentation

yican95 committed:
Update DG & PPP

rebstan97 committed:
Update DG

azhikai committed:
DeveloperGuide: Remove redundant points from use case

yican95 committed:
Merge pull request #282 from yican95/docs/PPP

v1.4 Minor fix and update PPP, UG & DG

azhikai committed:
team/azhikai: Added new point to documentation

rebstan97 committed:
Final updates to DG and PPP

HyperionNKJ committed:
Update PPP

azhikai committed:
Merge branch 'master' into feature/ug-dg-ppp

# Conflicts:
#	docs/DeveloperGuide.adoc
#	docs/UserGuide.adoc

rebstan97 committed:
Update PPP

rebstan97 committed:
Merge branch 'master' into docs/PPP

# Conflicts:
#	docs/DeveloperGuide.adoc
#	docs/UserGuide.adoc

azhikai committed:
UG/DG: Update to Yi Can's portion

rebstan97 committed:
Update DG

azhikai committed:
UG/DG: Correctly enclosed params in gray tag

rebstan97 committed:
Fix minor typos

HyperionNKJ committed:
Update PPP

rebstan97 committed:
Fix typo in PPP

HyperionNKJ committed:
Update PPP

HyperionNKJ committed:
Update PPP

azhikai committed:
UG/DG: Update callouts

rebstan97 committed:
Merge pull request #290 from rebstan97/docs/PPP

Update PPP and UGDG

HyperionNKJ committed:
Update PPP

azhikai committed:
Merge branch 'master' into feature/ug-dg-ppp

# Conflicts:
#	docs/DeveloperGuide.adoc
#	docs/UserGuide.adoc

azhikai committed:
UserGuide: Remove fullstop from block

yican95 committed:
Update UG & DG

yican95 committed:
Update PPP

yican95 committed:
DevelopGuide.adoc: Update figure number

azhikai committed:
DeveloperGuide: Hard-link contact page URL

azhikai committed:
DG: Fix alignment

HyperionNKJ committed:
Update PPP

HyperionNKJ committed:
Merge branch 'master' into docs/UGDGPPP

# Conflicts:
#	docs/DeveloperGuide.adoc
#	docs/UserGuide.adoc
#	src/main/java/seedu/restaurant/model/util/SampleDataUtil.java

HyperionNKJ committed:
Update sampledata

azhikai committed:
Merge pull request #294 from HyperionNKJ/docs/UGDGPPP

v1.4: Update UG, DG, PPP

yican95 committed:
Merge branch 'master' of https://github.com/yican95/main into docs/menu

azhikai committed:
DG: Fix alignment

yican95 committed:
SampleDataUtil.java: Update indentation

azhikai committed:
UG: Correct syntax error

azhikai committed:
Merge pull request #297 from yican95/docs/menu

v1.4 Update UG & DG

azhikai committed:
Merge branch 'master' into feature/ug-dg-ppp

# Conflicts:
#	docs/DeveloperGuide.adoc
#	docs/UserGuide.adoc

azhikai committed:
team/kaijun: Update PPP

azhikai committed:
DeveloperGuide: Update figure number

azhikai committed:
Fix KJ's PPP

azhikai committed:
Merge pull request #291 from AZhiKai/feature/ug-dg-ppp

Update UG/DG and PPP

azhikai committed:
UG: Fix formatting

azhikai committed:
Merge pull request #298 from AZhiKai/master

UG: Fix formatting

azhikai commented:
@dalessr On behalf the team, we'd like to extend our heartfelt gratitude to you for the help you've rendered us throughout the semester. I believe the team did our best, and hope that we were able to live up to your expectations.

Signing out for the last time. 😄

azhikai committed:
Update AboutUs.adoc

azhikai committed:
Cleanup images

azhikai committed:
Cleanup images

azhikai committed:
Cleanup images

azhikai committed:
Cleanup images

azhikai committed:
Cleanup images
